app/packages/state/src/hooks/useSchemaSettings.utils.ts

```diff
@@ -1,117 +1,158 @@
 import {
-  CLASSIFICATIONS_FIELD,
+  CLASSIFICATION_DISABLED_SUB_PATHS,
   CLASSIFICATION_FIELD,
-  DETECTIONS_FIELD,
+  DETECTION_DISABLED_SUB_PATHS,
   DETECTION_FIELD,
-  DYNAMIC_EMBEDDED_DOCUMENT_FIELD,
-  DYNAMIC_EMBEDDED_DOCUMENT_FIELD_V2,
-  FRAME_SUPPORT_FIELD,
+  DISABLED_FIELD_TYPES,
+  DISABLED_LABEL_FIELDS_VISIBILITY,
+  DISABLED_PATHS,
   Field,
+  GEOLOCATIONS_DISABLED_SUB_PATHS,
+  GEOLOCATION_DISABLED_SUB_PATHS,
   GEO_LOCATIONS_FIELD,
   GEO_LOCATION_FIELD,
+  HEATMAP_DISABLED_SUB_PATHS,
   HEATMAP_FIELD,
-  KEYPOINT_FILED,
-  LIST_FIELD,
-  POLYLINES_FIELD,
+  KEYPOINT_DISABLED_SUB_PATHS,
+  KEYPOINT_FIELD,
+  POLYLINE_DISABLED_SUB_PATHS,
   POLYLINE_FIELD,
-  REGRESSION_FILED,
-  RESERVED_FIELD_KEYS,
+  REGRESSION_DISABLED_SUB_PATHS,
+  REGRESSION_FIELD,
+  SEGMENTATION_DISABLED_SUB_PATHS,
   SEGMENTATION_FIELD,
+  TEMPORAL_DETECTION_DISABLED_SUB_PATHS,
   TEMPORAL_DETECTION_FIELD,
-  VALID_LABEL_TYPES,
-  VECTOR_FIELD,
 } from "@fiftyone/utilities";
-import { FRAME_NUMBER_FIELD, OBJECT_ID_FIELD } from "@fiftyone/utilities";
+
+export const isMetadataField = (path: string) => {
+  return path === "metadata" || path.startsWith("metadata.");
+};
 
 export const disabledField = (
   path: string,
   combinedSchema: Record<string, Field>,
-  groupField?: string
+  groupField?: string,
+  isFrameView?: boolean
 ): boolean => {
   const currField = combinedSchema?.[path] || ({} as Field);
-  const { ftype, embeddedDocType } = currField;
+  const { ftype } = currField;
   const parentPath = path.substring(0, path.lastIndexOf("."));
   const parentField = combinedSchema?.[parentPath];
```

Comment:
`combinedSchema` should always be defined. Is the optional chaining necessary?

Reply:
you are right. removed optional chain